plugins/field-colour/test/blocks_test.mocha.js

```diff
@@ -0,0 +1,272 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import fs from 'fs';
+import * as Blockly from 'blockly';
+
+import {javascriptGenerator} from 'blockly/javascript';
+import {dartGenerator} from 'blockly/dart';
+import {phpGenerator} from 'blockly/php';
+import {pythonGenerator} from 'blockly/python';
+import {luaGenerator} from 'blockly/lua';
+import {installAllBlocks} from '../src/index';
+import {assert} from 'chai';
+
+const blockJson = {
+  blocks: {
+    languageVersion: 0,
+    blocks: [
+      {
+        type: 'colour_picker',
+        id: '+g[iA}db.}w:CQV#k-p@',
+        x: 13,
```

Comment:
nit: sometimes in tests we recommend removing extraneous info from the serialization that isn't needed like IDs and coordinates, just to make things easier to read/maintain

Reply:
I removed IDs but kept coordinates because it sets the order of the blocks, which determines the order of the generated code.